deploy/deploy-FeeTaker.js

```diff
@@ -0,0 +1,25 @@
+const hre = require('hardhat');
+const { getChainId } = hre;
+
+module.exports = async ({ getNamedAccounts, deployments }) => {
+    console.log('running deploy script');
+    const chainId = await getChainId();
+    console.log('network id ', chainId);
+
+    const { deploy } = deployments;
```

Comment:
Just notify we have method with verify. It could be more compact.
```
const { deployAndGetContract } = require('@1inch/solidity-utils');
```

Reply:
Refactored, thanks!